app/visits/edit/template.hbs

```diff
@@ -226,6 +227,7 @@
                 </tr>
                 {{#each model.vitals as |vital|}}
                   <tr>
+                    <td>{{vital.modifiedBy}}</td>
```

Comment:
@adeolabadmus hold on... you might be right here.  modifiedBy used to be the users id, but it looks like it is now user display name.  Though we probably want the user who added, not who last modified, which would require an additional attribute.